pkg/gameserverallocations/allocator.go

```diff
@@ -82,6 +82,13 @@ const (
 	batchWaitTime         = 500 * time.Millisecond
 )
 
+const (
+	// Timeout for an individual Allocate RPC call can take
+	remoteAllocateTimeout = 10 * time.Second
```

Comment:
Also the client timeout needs to be set on the request context: https://grpc.io/blog/deadlines/#go

Reply:
I think I am setting it https://github.com/googleforgames/agones/blob/cd28618557d90d2a7c009f2f7420790286c26b37/pkg/gameserverallocations/allocator.go#L153
Unless you mean something else?